recipes/qpbo/recipe/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "qpbo" %}
+{% set version = "1.4.3" %}
+{% set sha256 = "3885119cddce875fa36221de4d14b577eb2f1696942274c7dcfcce904288d095" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: qpbo-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/qpbo/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/qpbo  # [unix]
```

Comment:
I guess we know this is true if we found the header below, right? No harm in having it, but it would be ok to drop.